apollo-runtime/build.gradle

```diff
@@ -32,11 +32,8 @@ android {
 }
 
 dependencies {
-  compile dep.jsr305
-  compile dep.supportAnnotations
-  compile dep.okHttp
-  compile dep.moshi
-  provided project(":apollo-api")
```

Comment:
Not sure that this is good idea, as we have to look now at 2nd place to find out what dependencies we are using for `:apollo-runtime` module